lib/components/narrative/metro/metro-itinerary.tsx

```diff
@@ -245,6 +250,17 @@ class MetroItinerary<Props> extends NarrativeItinerary {
       currency
     )
 
+    Object.keys(accessibilityScoreGradationMap || {}).forEach((key) => {
```

Comment:
Same question as in the other file -- is it not possible to do this inside the other loop? Also we should really consider putting this in a util file... It's pretty embarrassing to have duplicated code

Reply:
Not possible to do this inside the other loop because we need access to context. And I'm not sure about creating another util file just for these few lines. There is a lot of duplicated code in the gradation map thing, including some that is duplicated in both otp-ui and otp-rr. I think the best course of action here would be to move these icons into otp-ui and remove a lot of this code from otp-rr. Creating a util file now to slightly reduce duplication doesn't actually feel like it's actually solving the problem, but rather just checking a box.

Reply:
Would it be possible to move all the duplicated code to a utils file along with a big comment explaining how this should be moved?

Reply:
Alright, I moved it and the intl code into a util file. I wrote the icon one first and decided to just leave it in its own function, because why not.